client/http-client/Cargo.toml

```diff
@@ -28,8 +28,11 @@ jsonrpsee-test-utils = { path = "../../test-utils" }
 tokio = { version = "1.16", features = ["net", "rt-multi-thread", "macros"] }
 
 [features]
-default = ["tls"]
-tls = ["hyper-rustls/webpki-tokio"]
+default = ["native-tls"]
+native-tls = ["hyper-rustls/native-tokio", "__tls"]
```

Comment:
we can bikeshed on the naming of these features.

technically it only configures native vs webpki certificate store but without them no tls is configured.
I don't want to call them `native-cert` or something.

more accurate would be native-rustls and webpki-rustls but it's a bit hard to understand.